src/ds/ds-options.cpp

```diff
@@ -690,6 +690,21 @@ namespace librealsense
     void hdr_conditional_option::set(float value)
     {
         if (_hdr_cfg->is_config_in_process())
+            if( _hdr_cfg->is_enabled() )
+            {
+                // keep the index
+                auto _current_hdr_sequence_index = _hdr_cfg->get( RS2_OPTION_SEQUENCE_ID );
+                // In order to enable runtime exposure update in HDR mode we first send disable HDR, then set the value
+                // and at last enable HDR again.
+                _hdr_cfg->set( RS2_OPTION_HDR_ENABLED, 0, { 0, 1, 1 } );
```

Comment:
What is the {0,1,1} part?
Can't we use LibRS option instead?
I mean replace for example
`_hdr_cfg->set( RS2_OPTION_HDR_ENABLED, 0, { 0, 1, 1 } );`
with
`set_option(HDR_ENABLE, false)`
?

Reply:
I think that `set_option` is another level of abstract and that it is better to keep it here.

Reply:
The interface requires a range, we supply { 0, 1, 1 } means on or off